node/src/supportedRtpCapabilities.ts

```diff
@@ -335,6 +335,13 @@ const supportedRtpCapabilities: RtpCapabilities = {
 			preferredEncrypt: false,
 			direction: 'sendrecv',
 		},
+		{
+			kind: 'audio',
+			uri: 'http://www.webrtc.org/experiments/rtp-hdrext/playout-delay',
+			preferredId: 14,
+			preferredEncrypt: false,
+			direction: 'sendrecv',
+		},
```

Comment:
This is purely cosmetic but we always include the audio extension before the video extension.